flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/bootstrap.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+: ${HADOOP_PREFIX:=/usr/local/hadoop}
+
+$HADOOP_PREFIX/etc/hadoop/hadoop-env.sh
+
+rm /tmp/*.pid
+
+# installing libraries if any - (resource urls added comma separated to the ACP system variable)
+cd $HADOOP_PREFIX/share/hadoop/common ; for cp in ${ACP//,/ }; do  echo == $cp; curl -LO $cp ; done; cd -
+
+# kerberos client
+sed -i "s/EXAMPLE.COM/${KRB_REALM}/g" /etc/krb5.conf
```

Comment:
`EXAMPLE.COM` is used in several places, is there any way we can set this in a single place? (for example with search&replace if necessary)

Reply:
`EXAMPLE.COM` is pretty buch the placeholder for this and could be replaced with a different realm in `bootstrap.sh`. But the default is just to still use `EXAMPLE.COM`. I could rename this `TEMPLATE.URL` if you want. 😅

Reply:
yeah nvm, I doubt introducing a placeholder really fixes things :/